Jenkinsfile

```diff
@@ -63,6 +63,13 @@ node {
     // test report on the basis of suite and see report from Allure report plugin &
     // see test code coverage report from Coverage report Plugin only when suite choice_test_all and choice_test_python
     stage ("Test Report"){
+        // Preparing Database setup
+        sh "PGPASSWORD=postgres psql -U postgres -h localhost -f ${workspace_dir}/src/sql/foglamp_ddl.sql postgres"
```

Comment:
Looks good! seems keeping it in `.sh` file and having **jenkins** as var ${db_user} will allow to setup on any ubuntu machine with any username